docs/topics/strings.md

```diff
@@ -132,4 +141,36 @@ use the following syntax:
 val price = """
 ${'$'}_9.99
 """
-```
\ No newline at end of file
+```
+## String formatting
```

Comment:
Should have blank lines around, before and after

Reply:
Done it. Thanks.

Reply:
What about one after? Would be nicer IMO

Reply:
Yes, please place empty lines before and after headers, as per our [style guide](https://docs.google.com/document/d/1mUuxK4xwzs3jtDGoJ5_zwYLaSEl13g_SuhODdFuh2Dc/edit#heading=h.uf30glbmdezp).

Reply:
Done.